build/webpack.server.conf.js

```diff
@@ -1,17 +1,13 @@
-var webpack = require('webpack');
 var merge = require('webpack-merge');
-var path = require('path');
 var baseWebpackConfig = require('./webpack.base.conf');
 var nodeExternals = require('webpack-node-externals');
 var VueSSRServerPlugin = require('vue-server-renderer/server-plugin');
 var FriendlyErrorsPlugin = require('friendly-errors-webpack-plugin');
 
-var noop = path.resolve(__dirname, 'no-op.js');
-
 module.exports = merge.smart(baseWebpackConfig, {
 	entry: './src/server-entry.js',
 	target: 'node',
-	devtool: '#source-map',
+	devtool: 'eval',
```

Comment:
If your running server debug processes on code outside of vue components it would be helpful to change this back. But for dev purposes I'm fine setting to eval.

That said, I don't think we should leave this as 'eval' for the compiled build that we deploy. Seems like we should switch back to 'source-map' for that... maybe just a prod toggle like in other places...

Reply:
Similar to webpack.client.base. where we import the config then run this...
`config.build.productionSourceMap ? '#source-map' : false,` <-making false 'eval' instead.

Reply:
`config.build.productionSourceMap ? '#source-map' : false`

I don't believe this code works at all. (See my new commit for changes) I tested and config.build.productionSourceMap is always true. It is a key in the config file and it is always true. For the dev server and builds the false condition of the ternary never ran. I changed this check to be isProd and based on the node env instead. This works as expected. 

Also, I could not find any reference to '#source-map' the WebPack 4 docs reference that option as 'source-map'